.azure-pipelines/templates/automation_test.yml

```diff
@@ -32,23 +32,31 @@ steps:
       echo fullTest: ${{ parameters.fullTest }}
       echo module: ${{ parameters.module }}
       echo Build.Reason: $(Build.Reason)
+      series_modules="appservice botservice cloud network azure-cli-core azure-cli-telemetry"
```

Comment:
`series_modules` are defined twice, one here, one in `automation_full_test.py`. Maybe we can unify them to reduce duplication.

Reply:
Modify series_modules to serial_modules to ensure consistency.